### PR TITLE
Made small changes to make the README and CONTRIBUTING files easier to follow

### PR DESCRIPTION
**Pull request by BurkHufnagel**
Made small changes in punctuation, word selection, and word order to make it easier to read and understand the README and CONTRIBUTING 'md' files.

### CODE REVIEW

**Review by markjacksonfishing — APPROVED** [review]
🙂

**Review by oleg-nenashev — APPROVED** [review]
Thanks! All changes look good to me.

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Review by MarkEWaite — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,58 +2,57 @@
...
-3. Install the development tools. In order to develop Jenkins, you need the following tools:
+3. Install the necessary development tools. In order to develop Jenkins, you need the following:
   * Java Development Kit (JDK) 8 or 11.
-    In Jenkins project we usually use [OpenJDK](http://openjdk.java.net/) or [AdoptOpenJDK](https://adoptopenjdk.net/), but you can use other JDKs as well.
+    In the Jenkins project we usually use [OpenJDK](http://openjdk.java.net/) or [AdoptOpenJDK](https://adoptopenjdk.net/), but you can use other JDKs as well.
     * For JDK 11 there might be some compatibility issues in developer tools,
       please see [this page](https://wiki.jenkins.io/display/JENKINS/Java+11+Developer+Guidelines#Java11DeveloperGuidelines-Knowndevelopertoolsissues) for more info.
       If you hit a new issue, please report it with a `java11-devtools-compatibility` label in our issue tracker.
-  * Maven 3.5.4 or above. You can [download maven].
+  * Maven 3.5.4 or above. You can [download maven] here.
```
Comment (daniel-beck) [inline CONTRIBUTING.md:17]:
```suggestion
  * Maven 3.5.4 or above. You can [download maven here].
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,58 +2,57 @@
...
 In the Jenkins issue tracker we mark such issues as `newbie-friendly`.
-You can find them
-using this query for [newbie friendly issues].
+You can find them by using this query (check the link) for [newbie friendly issues].
 
 ## Building and Debugging
 
-The build flow for Jenkins core is built around Maven.
-There is a description of the [building and debugging process].
+The core of Jenkins' build flow is built around Maven.
```
Comment (daniel-beck) [inline CONTRIBUTING.md:30]: Introduces a semantic change (or an ambiguity, depending on how you read it): "Jenkins core" is a fixed term we use to refer to Jenkins without plugins.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,58 +2,57 @@
...
+Here is a description of the [building and debugging process].
```
Comment (daniel-beck) [inline CONTRIBUTING.md:31]: Weird to see "here" when the "here" isn't clickable.

```suggestion
[Here is a description of the building and debugging process].
```

might be a bit much but still better? Maybe?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -93,42 +92,42 @@ cd war; yarn test
...
 
-1. Commit changes and push them to your fork on GitHub.
+1. Commit your changes and push them to your fork on GitHub.
 It is a good practice is to create branches instead of pushing to master.
-2. In GitHub Web UI click the _New Pull Request_ button
-3. Select `jenkinsci` as _base fork_ and `master` as `base`, then click _Create Pull Request_
-  * We integrate all changes into the master branch towards the Weekly releases
-  * After that the changes may be backported to the current LTS baseline by the LTS Team.
-    Read more about the [backporting process]
+2. In GitHub Web UI, click the _New Pull Request_ button.
```
Comment (daniel-beck) [inline CONTRIBUTING.md:101]:
```suggestion
2. In the GitHub Web UI, click the _New Pull Request_ button.
```

File: README.md [hunk 5]
```diff
@@ -62,7 +62,7 @@ Jenkins Code of Conduct can be found [here](https://www.jenkins.io/project/condu
 
 # Adopters
 
-Jenkins is used by millions of users and thousands of companies.
+Jenkins is used by millions of users, and thousands of companies.
```
Comment (daniel-beck) [inline README.md:65]: I don't think this is correct.

**Review by daniel-beck — CHANGES_REQUESTED** [review]
Requesting changes for the rewrite of "Jenkins core" at least.

File: CONTRIBUTING.md (hunk 1 above)
Comment (BurkHufnagel) [reply CONTRIBUTING.md]: Done -- also capitalized Maven.

File: CONTRIBUTING.md (hunk 2 above)
Comment (BurkHufnagel) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 3 above)
Comment (BurkHufnagel) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 4 above)
Comment (BurkHufnagel) [reply CONTRIBUTING.md]: Done

File: README.md (hunk 5 above)
Comment (BurkHufnagel) [reply README.md]: Undone.

**Comment by BurkHufnagel** [comment]
I think I've addressed all the concerns raised.

File: README.md [hunk 6]
```diff
@@ -62,7 +62,7 @@ Jenkins Code of Conduct can be found [here](https://www.jenkins.io/project/condu
 
 # Adopters
 
-Jenkins is used by millions of users and thousands of companies.
+Jenkins is used by millions of user and thousands of companies.
```
Comment (daniel-beck) [inline README.md:65]:
```suggestion
Jenkins is used by millions of users and thousands of companies.
```

**Review by daniel-beck — DISMISSED** [review]
Requesting just a small change.

Personally I'm also surprised at the addition of the many commas, my English writings tend to get them removed in review.

**Comment by oleg-nenashev** [comment]
Thanks for your contribution! If this is a Hacktoberfest pull request, please let us know so that we can label it properly and facilitate reviews. Check out [this page](https://www.jenkins.io/events/hacktoberfest/) for more information about Jenkins in Hacktoberfest: guidelines, featured projects, etc. 

[image](https://www.jenkins.io/events/hacktoberfest/)

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by halkeye** [comment]
As a general rule, "here" is considered bad practice as its confusing for screen readers. The idea is that you linkify the topic

https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/ for some more details

Its probably fine in this PR, but something to remember.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -2,58 +2,57 @@
 
 This page provides information about contributing code to the Jenkins core codebase.
 
-:exclamation: There's a lot more to the Jenkins project than just code. For information on contributing to the Jenkins project overall, check out [Participate].
+:exclamation: There's a lot more to the Jenkins project than just code. For information on contributing to the Jenkins project overall, here's a page on how to [Participate].
```
Comment (andreamussap) [inline CONTRIBUTING.md:5]: 'For information on contributing to the Jenkins project overall, here's a page on how to [Participate].'  - **suggestion**: For more information on the ways that you can contribute to the Jenkins project, see [Participate].

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -2,58 +2,57 @@
...
 1. Fork the repository on GitHub
 2. Clone the forked repository to your machine
-3. Install the development tools. In order to develop Jenkins, you need the following tools:
+3. Install the necessary development tools. In order to develop Jenkins, you need the following:
   * Java Development Kit (JDK) 8 or 11.
-    In Jenkins project we usually use [OpenJDK](http://openjdk.java.net/) or [AdoptOpenJDK](https://adoptopenjdk.net/), but you can use other JDKs as well.
+    In the Jenkins project we usually use [OpenJDK](http://openjdk.java.net/) or [AdoptOpenJDK](https://adoptopenjdk.net/), but you can use other JDKs as well.
     * For JDK 11 there might be some compatibility issues in developer tools,
       please see [this page](https://wiki.jenkins.io/display/JENKINS/Java+11+Developer+Guidelines#Java11DeveloperGuidelines-Knowndevelopertoolsissues) for more info.
       If you hit a new issue, please report it with a `java11-devtools-compatibility` label in our issue tracker.
```
Comment (andreamussap) [inline CONTRIBUTING.md:16]: Suggestion -> If you find a new issue...

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -2,58 +2,57 @@
...
-  * Maven 3.5.4 or above. You can [download maven].
+  * Maven 3.5.4 or above. You can [download Maven here].
```
Comment (andreamussap) [inline CONTRIBUTING.md:17]: I'd add the link to download Maven in the instruction itself, e.g. - [Maven 3.5.4 or above](link).

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -2,58 +2,57 @@
...
   * Any IDE which supports importing Maven projects.
-  * Install [NodeJS](https://nodejs.org/en/). **Note:** only needed to work on the frontend assets found on the `war` module.
-    * Frontend tasks are ran using [yarn](https://yarnpkg.com/lang/en/). Run `npm install -g yarn` to install it.
+  * Install [NodeJS](https://nodejs.org/en/). **Note:** only needed to work on the frontend assets found in the `war` module.
+    * Frontend tasks are run using [yarn](https://yarnpkg.com/lang/en/). Run `npm install -g yarn` to install it.
 4. Setup your development environment as described in [Preparing for Plugin Development]
```
Comment (andreamussap) [inline CONTRIBUTING.md:21]: Set up your development environment

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -145,13 +144,13 @@ just submit a pull request.
 * [Jenkins Contribution Landing Page](https://jenkins.io/participate/)
 * [Jenkins IRC Channel](https://jenkins.io/chat/)
 * [Beginners Guide To Contributing](https://wiki.jenkins.io/display/JENKINS/Beginners+Guide+to+Contributing)
-* [List of newbie-friendly issues in the core](https://issues.jenkins-ci.org/issues/?jql=project%20%3D%20JENKINS%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20component%20%3D%20core%20AND%20labels%20in%20(newbie-friendly))
+* [List of newbie-friendly issues in the core](https://issues.jenkins-ci.org/issues/?jql=project%20%3D%20JENKINS%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20coavenponent%20%3D%20core%20AND%20labels%20in%20(newbie-friendly))
```
Comment (MarkEWaite) [inline CONTRIBUTING.md:147]: Proposed change will break the query.  Need to revert this change
```suggestion
* [List of newbie-friendly issues in the core](https://issues.jenkins-ci.org/issues/?jql=project%20%3D%20JENKINS%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20component%20%3D%20core%20AND%20labels%20in%20(newbie-friendly))
```

**Comment by MRamonLeon** [comment]
Seems to be some suggestions still pending to address, I remove the _ready-for-merge_ label.

File: CONTRIBUTING.md (hunk 8 above)
Comment (oleg-nenashev) [reply CONTRIBUTING.md:16]:
```suggestion
      If you find a new issue, please report it with a `java11-devtools-compatibility` label in our issue tracker.
```

File: CONTRIBUTING.md (hunk 7 above)
Comment (oleg-nenashev) [reply CONTRIBUTING.md:5]:
```suggestion
:exclamation: There's a lot more to the Jenkins project than just code. For more information on the ways that you can contribute to the Jenkins project, see [Participate].
```

@andreashouben GitHub suggestions are quite handy when it comes to reviews and suggestions

File: CONTRIBUTING.md (hunk 10 above)
Comment (oleg-nenashev) [reply CONTRIBUTING.md:21]:
```suggestion
4. Set up your development environment as described in [Preparing for Plugin Development]
```

File: CONTRIBUTING.md (hunk 9 above)
Comment (oleg-nenashev) [reply CONTRIBUTING.md:17]: IMHO it is an overkill here, especially since we have a download link on the same line

**Comment by oleg-nenashev** [comment]
We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by oleg-nenashev** [comment]
Thanks all!
